### PR TITLE
Added missing field parsing

### PR DESCRIPTION
**Pull request by IvanivOleg**
Added parsing of missing "state" field

### CODE REVIEW

File: slack-base/src/main/java/com/hubspot/slack/client/models/interaction/BlockActionsIF.java [hunk 1]
```diff
@@ -26,6 +26,8 @@
   @JsonProperty("actions")
   List<BlockElementAction> getElementActions();
 
+  ViewStateValuesPayload getState();
```
Comment (omotnyk) [inline slack-base/src/main/java/com/hubspot/slack/client/models/interaction/BlockActionsIF.java:29]: Ideally, we'd like to rename this class to something more generic as it doesn't represent solely "View" state anymore. The only drawback is that this is going to be a breaking change. Thoughts?

File: slack-base/src/test/resources/block_actions.json [hunk 2]
```diff
@@ -75,5 +75,32 @@
...
+                "text":" 2",
+                "emoji":true
+              },
+              "value":"11"
+            }
+          ]
+        }
+      }
+    }
+  }
```
Comment (omotnyk) [inline slack-base/src/test/resources/block_actions.json:105]: 👍

File: slack-base/src/main/java/com/hubspot/slack/client/models/interaction/BlockActionsIF.java (hunk 1 above)
Comment (IvanivOleg) [reply slack-base/src/main/java/com/hubspot/slack/client/models/interaction/BlockActionsIF.java]: Fixed

**Review by omotnyk — APPROVED** [review]
🚀

**Pull request merged**